_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: news
+title: Team TLC Wins Student Robotics 2019
+---
+
+{% include figure.html
+           src="/images/content/news/sr2019/sr2019-photo.jpg"
+           caption="All the teams from SR2019" %}
+
+
+That's a wrap on Student Robotics 2019! Our teams have been working hard since November to design, build, and test their robots; finally seeing the fruits of their work on the competition weekend. It was great to see our competitors quickly adapting their robots, from the physical design to the software, to overcome unforeseen challenges; giving themselves every small advantage towards claiming victory.
+
+The Challenge: Caldera
+----------------------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/arena.jpg"
+           caption="The arena with a raised volcano area" %}
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/tlc-launcher.jpg"
+           caption="Team TLC's token launcher" %}
+
+           
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/swi.jpg"
+           caption="Team SWI's volcano climbing robot in the Caldera" %}
+
+Our game this year, Caldera, challenged teams to control scoring zones with tokens while navigating an arena with raised platforms. Twenty-five scoring zones were arranged in a grid, varying in value from two points for each of the outermost 16 zones, a ring of eight on a raised “volcano” worth seven points apiece, to the the central “caldera” which was worth thirty points.
```

Comment:
Should we also mention the robot-position bonus here?

I'm torn because it's a very necessary detail in order to understand both the match scores from the arena configurations, plus it indicates an orthogonal part of the challenge, however it's non-trivial to explain and I'm not sure it adds a lot to the high-level view of the game.

Reply:
"Parking your robot in a zone tripled the points of that zone and the 4 adjacent zones."

Is that explained simply enough?